unity-renderer/Assets/DCLPlugins/ECS7/ProtocolBuffers/Editor/ProtobufEditor.cs

```diff
@@ -50,7 +50,8 @@ static ProtobufEditor()
 
         // Use this parameter when you want a fixed version of the @dcl/protocol, otherwise leave it empty
         //private const string FIXED_NPM_PACKAGE_LINK = "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-2611997102.commit-8e362ff.tgz";
-        private const string FIXED_NPM_PACKAGE_LINK = "";
+        // REVERT THIS FILE BEFORE MERGE! (requires @dcl/protocol PR merge)
+        private const string FIXED_NPM_PACKAGE_LINK = "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-2818365550.commit-b11aed2.tgz";
```

Comment:
This URL shouldn't be in main or dev IMO. It should download the last version if clicked in the option in Unity